index.html

```diff
@@ -89,7 +89,90 @@ <h2 class="heading" id="job">Software Developer</h2>
     <br>
     <br>
    <img src="resources/work.png" id="last-img" alt="work">
-    </div>
-
+   <br>
+   <br>
+   <br>
+   <section class="work">
```

Comment:
Good job for using <section> 👍 

Inside this section, you have to organize your code like this: for each project make it on a `div`  and use flexbox to align it.

Your code will be like this:
```
 <section class="work">
     <div> first project (h2,a p ...) </div>
     <div> 2nd project </div>
</section>
```

Reply:
:heavy_check_mark: Good job at handling this issue